helm/silence-operator/values.yaml

```
@@ -55,3 +55,6 @@ resources:
 
 rbac:
   create: true
+
+crds:
+  install: true
```

Where does the `description` in the schema come from?
I'd expect to see it directly in the values file as well.

```suggestion
  # Whether to install the Silence CRD with this chart.
  install: true
```

---

This was AI generated